kubernetes/kubernetes/src/main/java/org/arquillian/cube/kubernetes/impl/SessionCreatedListener.java

```diff
@@ -122,6 +127,7 @@ public void run() {
 
             if (!configuration.isEnvironmentInitEnabled() || waitStrategy.await()) {
                 displaySessionStatus(session);
+                afterStartEvent.fire(new AfterStart(session));
```

Comment:
start & stop event classes are inheriting from SessionCreatedEvent. So if we use it directly then your observer is calling for firing of start & stop as well. I don't want to invoke it for all fires of start & stop.